tools/c7n_azure/c7n_azure/resources/storage.py

```diff
@@ -24,3 +28,52 @@ class resource_type(ArmResourceManager.resource_type):
         client = 'StorageManagementClient'
         enum_spec = ('storage_accounts', 'list', None)
         diagnostic_settings_enabled = False
+
+
+@Storage.action_registry.register('set-network-rules')
+class StorageSetNetworkRulesAction(AzureBaseAction):
+
+    schema = type_schema(
+        'set-network-rules',
+        required=['default-action'],
+        **{
+            'default-action': {'enum': ['Allow', 'Deny']},
+            'bypass': {'type': 'string'},
```

Comment:
nit: does it make sense to use array of strings for `bypass`?

Reply:
good idea